### DISCUSSION TITLE
How to create a server using mods and plugins together?

### DISCUSSION THREAD

**Discussion by marlonangeli** [itzg/docker-minecraft-server#3012] [Q&A]
I'm trying to use a modpack from CurseForge so I use MOD_PLATFORM: AUTO_CURSEFORGE, that works fabulous. But I want to add some plugins, using Spigot or Bukkit or passing via volumes using `/plugins`.

Here is the link of the repo: https://github.com/marlonangeli/minecraft-server

I want to add [OpenLogin](https://www.spigotmc.org/resources/openlogin-1-7x-1-21x.57272/) plugin, I try downloading and using volumes, downloading via `SPIGET_RESOURCES` but none works :/

Any help??

**Comment by itzg** [comment]
To use plugins along with mods you'll need to use a hybrid server type

https://docker-minecraft-server.readthedocs.io/en/latest/types-and-platforms/server-types/hybrids/

...however there's not a way to combine that with CurseForge modpacks since the modpack dictates the modloader. With the existing content from the modpack in place I suppose you could switch out the `TYPE` variable to run the hybrid server type you choose.
